app/components/release_list_component.html.erb

```diff
@@ -176,70 +176,7 @@
         </div>
       <% end %>
     <% end %>
-
-    <%= render SectionComponent.new(style: :titled, title: "Previous Releases") do %>
-      <% if previous_releases.exists? %>
-        <%= render TableComponent.new(columns: release_table_columns, size: :default) do |table| %>
-          <% previous_releases.each do |release| %>
-            <% release = ReleasePresenter.new(release, view_context) %>
-            <% table.with_row do |row| %>
-              <% row.with_cell(style: "") do %>
-                <div class="flex space-x-2">
-                  <%= render BadgeComponent.new(**release.release_status) %>
-                  <%= render(release.hotfix_badge) unless release.hotfix_badge.nil? %>
-                  <%= render(release.scheduled_badge) if release.automatic? %>
-                </div>
-              <% end %>
-
-              <% row.with_cell(style: "font-medium") do %>
-                <% if release.completed_at %>
-                  <%= render BadgeComponent.new(text: release.release_version, kind: :badge) %>
-                <% else %>
-                  <%= render BadgeComponent.new(text: version_in_progress(release.release_version), kind: :badge) %>
-                <% end %>
-              <% end %>
-
-              <% row.with_cell do %>
-                <%= render BadgeComponent.new(text: release.release_branch, kind: :badge) %>
-              <% end %>
-
-              <% if reldex_defined? %>
-                <% row.with_cell do %>
-                  <% if (reldex = release.reldex) %>
-                    <%= render Reldex::StatusComponent.new(release:, reldex_score: reldex) %>
-                  <% else %>
-                    <%= NOT_AVAILABLE %>
-                  <% end %>
-                <% end %>
-              <% end %>
-
-              <% row.with_cell do %>
-                <div><%= release.interval %></div>
-              <% end %>
-
-              <% row.with_cell(style: "text-right w-2") do %>
-                <%= render ButtonComponent.new(
-                  scheme: :supporting,
-                  label: "Go to release",
-                  options: release_path(release),
-                  size: :sm,
-                  authz: false,
-                  type: :link) do |b|
-                  b.with_icon("arrow_right.svg")
-                end %>
-              <% end %>
-            <% end %>
-          <% end %>
-        <% end %>
-      <% else %>
-        <%= render EmptyStateComponent.new(
-          title: "No previous releases",
-          text: "There are no other releases for this train yet.",
-          banner_image: "archive_x.svg",
-          type: :subdued
-        ) %>
-      <% end %>
-    <% end %>
+    <%= helpers.turbo_frame_tag "previous_releases", src: previous_releases_app_train_releases_path(params[:app_id], params[:train_id]) %>
```

Comment:
On first load, there's a lag before the previous releases render. That's because the page makes a separate turbo frame call after the initial load. This is fine, but if we do this, then we should put a loading indicator (see `EnhancedTurboFrameComponent` or `LoadingIndicatorComponent`) to suggest that some content will render there.